security-whitepaper/README.md

```diff
@@ -1,14 +1,80 @@
+spelling, clarification in language, translations, etc.  When these occur 
+they are considered minor changes to the overall content and will not warrant
+ the regeneration of the PDF.
+
+When significant changes to the intent, content, or numerous minor changes
+occur, the CNSWP working group will assess and determine if a new major version
+of the PDF needs published.  When this decision is made, the markdown content
+will be converted to text document and sent to the CNCF technical writers to
+create the PDF.  The PDF will then be published back into the repository
+annotating the new version, updating the links in the README.md accordingly.
+
+## Original design decisions 
+The CNSWP's creation occurred using the below general design decisions which 
+should be considered when updating the content.  
+* Avoid identifying specific projects and products.  Use general terms
+  identifying the capabilities of the projects that meet the content under
+discussion.  Examples include, but are not limited to:
+  * Orchestrator instead of Kubernetes
+  * Secrets store instead of Vault
+  * Policy-based control instead of OPA
+* Consider if the content already exists elsewhere.  Provide references to
+  comprehensive information on a topic rather than re-writing the content.  This
+not only allows us to provide summarization of complex topics, but also exposes
+the audience to other avenues of information for which they may be unaware.
+* Determine if the content is better suited as it's own document, such as a
+  how-to, blog, or whitepaper of itself.  The CNSWP is intended to be an initial
+starting point for the community in understanding the intricacies of a secure
+cloud native workload and architecture.  Detailed information should be
+referenced elsewhere and maintained distinctly so the reader may choose to
+research further if it suits their needs.
+* Identify if the proposed update will require an update to the executive
+  summary.  This is not likely to be the case for most updates, however should
+still be considered.  The executive summary is intended to be kept at no more
+than 3 printable pages.  The executive summary should be a brief, high level
+overview of the paper's contents for those without the immediate time to read
+the length of the document.
 
 Links:
-* [Managed version in Markdown](https://github.com/cncf/sig-security/blob/master/security-whitepaper/cloud-native-security-whitepaper.md)
-* [Posted PDF first version](https://github.com/cncf/sig-security/blob/master/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf)
-* [CNCF Blog Post](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/)
-* [Kubernetes Blog Post](https://kubernetes.io/blog/2020/11/18/cloud-native-security-for-your-clusters/)
+* [Managed version in  Markdown](https://github.com/cncf/sig-security/blob/master/security-whitepaper/cloud-native-security-whitepaper.md)
```

Comment:
Should we specify how the status of the md file should be managed when starting to make changes there -- for example an easy way to know that it is a draft that hasn't been published as pdf yet. If so where is a good place to mention that? May be spell that out in the `versioning and publishing` section?